qa/multi-cluster-search/src/test/resources/rest-api-spec/test/multi_cluster/20_info.yml

```diff
@@ -152,7 +152,7 @@
   - do:
       cluster.remote_info: {}
 
-  - is_false: remote1.skip_unavailable
+  - is_true: remote1.skip_unavailable
```

Comment:
I guess these yaml tests are run only against the latest versions and never cross version?

Reply:
I think so? The build hasn't been failing on bwc tests. If it does at some point, we can just remove this first test block that tests the default and require the value to be explicitly set in the yaml tests.

Reply:
This does make me wonder if we are lacking tests. Do we have any that verify the behavior with clusters from different versions?